src/config.c

```diff
@@ -887,6 +887,18 @@ void RSConfig_AddToInfo(RedisModuleInfoCtx *ctx) {
   RedisModule_InfoAddFieldLongLong(ctx, "min_phonetic_term_length", RSGlobalConfig.minPhoneticTermLen);
 }
 
+void DialectsGlobalStats_AddToInfo(RedisModuleInfoCtx *ctx) {
+  RedisModule_InfoAddSection(ctx, "dialect_statistics");
+  RedisModule_InfoAddFieldULongLong(ctx, "min_dialect_version", MIN_DIALECT_VERSION);
+  RedisModule_InfoAddFieldULongLong(ctx, "max_dialect_version", MAX_DIALECT_VERSION);
```

Comment:
The min and max are actually redundant (can be derived from the list of used dialects which follows)
and since these are used for testing - I prefer to remove them (and make tests extract min and max from the dialect list)
It would also be aligned with the `dialect_stats` in `FT.INFO`